frontend/src/widgets/genericarea.js

```diff
@@ -108,6 +108,7 @@ export class GenericAreaWidget extends React.Component {
           areaCharts.push(
             <ChartArea
               data={chartData}
+              key={areaCharts.length}
```

Comment:
I was getting errors in the UI that there were not unique keys for the `areaCharts` list items. I used the length as something I know would be updated dynamically, but we should use something else here. I was just trying to hack my way to a solution first. Thoughts?

Reply:
You could just use an increasing index?

Reply:
Updated.